docker-compose.yml

```diff
@@ -34,11 +34,19 @@ services:
 
   db:
     image: postgres:9.5
+    ports:
+      - 5432:5432
     environment:
       POSTGRES_DB: pah-fm
       POSTGRES_USER: pah-fm
       POSTGRES_PASSWORD: pah-fm
 
+  adminer:
```

Comment:
What's the purpose of adding another container with (admittedly lightweight) image when it's not required for the project to operate?
Adminer's purpose is to act as an easy interface for database but this approach enforces it for every developer even if someone's using *any* other client, such as dbeaver, built-in pycharm DB tools or even raw PSQL shell.

If someone wants to interact with the database via adminer then she should spin it up with simple `docker run` command, no need to add it as a dependency.

Reply:
I think it's helpful for development, but I agree it will be really easy to add it directly from terminal. Let me remove this